examples/mysql-batch-execute-operation/mysql_batch_execute_operation.md

```diff
@@ -1,23 +1,18 @@
-# MySQL client - Batch execution
+# Database Access - Batch execution
 
 This BBE demonstrates how to use the MySQL client to execute a batch of DDL/DML operations. 
 
+>> Tip: Checkout [`ballerinax/mssql`](https://central.ballerina.io/ballerinax/mssql), [`ballerinax/postgresql`](https://central.ballerina.io/ballerinax/postgresql), [`ballerinax/oracledb`](https://central.ballerina.io/ballerinax/oracledb), [`ballerinax/java.jdbc`](https://central.ballerina.io/ballerinax/java.jdbc) database clients.
```

Comment:
```suggestion
>> Tip: Checkout [`ballerinax/mssql`](https://central.ballerina.io/ballerinax/mssql), [`ballerinax/postgresql`](https://central.ballerina.io/ballerinax/postgresql), [`ballerinax/oracledb`](https://central.ballerina.io/ballerinax/oracledb), [`ballerinax/java.jdbc`](https://central.ballerina.io/ballerinax/java.jdbc) for other database clients.
```

Reply:
Fixed in [0dadc8c](https://github.com/ballerina-platform/ballerina-distribution/pull/3648/commits/0dadc8c0c8f1c92be35c13306bc45f51b1bbf2ec)